lib/internal/Magento/Framework/Filesystem/Test/Unit/Directory/PathValidatorTest.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * Unit Test for \Magento\Framework\Filesystem\Directory\PathValidator
+ *
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Framework\Filesystem\Test\Unit\Directory;
+
+use Magento\Framework\Filesystem\Directory\WriteInterface;
+use Magento\Framework\Filesystem\DriverInterface;
+
+class PathValidatorTest extends \PHPUnit\Framework\TestCase
+{
+    /**
+     * \Magento\Framework\Filesystem\Driver
+     *
+     * @var \PHPUnit_Framework_MockObject_MockObject
+     */
+    protected $driver;
+
+    /**
+     * @var \Magento\Framework\Filesystem\Directory\PathValidator
+     */
+    protected $pathValidator;
+
+    /**
+     * Set up
+     */
+    protected function setUp()
+    {
+        $this->driver = $this->createMock(\Magento\Framework\Filesystem\Driver\File::class);
+        $this->pathValidator = new \Magento\Framework\Filesystem\Directory\PathValidator(
+            $this->driver
+        );
+    }
+
+    /**
+     * Tear down
+     */
+    protected function tearDown()
+    {
+        $this->driver = null;
```

Comment:
Why it’s needed? Never saw that variables are set to null in tear down method

Reply:
I copied over the implementation of of ReadTest and WriteTest, where they clean up the driver mockup pointer in the tearDown method. I guess historically this is for cleaning up the resources allocated in the constructor of mocking target.

While I did some test with latest phpunit, it doesn't initialize the mocking target class at all, so I don't think the clean-up is needed either.